src/api/l10n-translations/ja/l10n_reactor

```diff
@@ -0,0 +1,37 @@
+-- vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 syntax=lua
+--
+-- Big Reactors control API - translation file
+--
+-- Language: Japanese
+--
+-- (c) 2016, random-host.com
+
+t_title = 'リアクターコントロルのプログラム'
+t_consoleRunning = '停止する場合：何でもキイをおせる。かわりに、[CTRL]+[T] をおせてる。'
+t_fuelLevelsTitle = '燃料のレベル：'
+t_fuelLevelsFuel = '燃料'
+t_fuelLevelsWaste = '屑'
+t_temperatureTitle = '温度:'
+t_temperatureCasing = '筐体:'
+t_temperatureFuel  = '炉心:  '
+t_controlRodLevelsTitle = '制御棒のレベル:'
+t_controlRodLevelsRod = '制御棒'
+t_controlRodLevelsDepth = '奥行き'
+t_bufferTitle = 'バッファー:'
+t_outputTitle = '産出'
+t_statusTitle = '現状は'
+t_statusOnline = 'オンライン'
+t_statusOffline = 'オフライン'
+t_messagesTitle = 'メッセジ:'
+t_messagesOnline = '発電する...'
+t_messagesOffline = '待機...'
+t_messagesNoFuel = '燃料ではない'
+t_messagesBufferFull = 'Buffer full'
+t_startButtonTitle = 'Status Control:'
+t_startButtonStart = ' start '
+t_startButtonStop = ' stop  '
```

Comment:
Is there a specific reason why you didn't translate these 4 strings?

Reply:
Whoops, I missed those. My bad!